ccmlib/scylla_cluster.py

```diff
@@ -98,8 +98,7 @@ def start_nodes(self, nodes=None, no_wait=False, verbose=False, wait_for_binary_
             wait_for_binary_proto = self.force_wait_for_cluster_start
         if wait_other_notice is None:
             wait_other_notice = self.force_wait_for_cluster_start
-        self.debug(f"start_nodes: no_wait={no_wait} wait_for_binary_proto={wait_for_binary_proto} wait_other_notice={wait_other_notice} force_wait_for_cluster_start={self.force_wait_for_cluster_start}")
-        self.started=True
+        self.debug(f"start_nodes: no_wait={no_wait} wait_for_binary_proto={wait_for_binary_proto} wait_other_notice={wait_other_notice} force_wait_for_cluster_start={self.force_wait_for_cluster_start} scylla_mode={self.scylla_mode} default_wait_other_notice_timeout={self.default_wait_other_notice_timeout} default_wait_for_binary_proto={self.default_wait_for_binary_proto}")
```

Comment:
don't we have enough prints showing the modes ?

why do we need it on each node starting ?

Reply:
We have a printout in dtest_setup.py `init_default_config` and or printed the apparently correct mode:
```
06:51:00,723 790     dtest_setup                    DEBUG    dtest_setup.py      :694  | test_cluster_expansion_with_cdc[Single_cluster]: Scylla mode is 'debug'
```

But timeout==420 is an evidence for the mode not set up properly in `ScyllaCluster.__init__`

Reply:
I can move the printout to the constructor so it's printed once per cluster instantiation.

Reply:
or print the extra information only `if not self.started`